cypress/e2e/cloud/globalHeader.test.ts

```diff
@@ -3,26 +3,6 @@ import {makeQuartzUseIDPEOrgID} from 'cypress/support/Utils'
 describe('change-account change-org global header', () => {
   let idpeOrgID: string
 
-  const interceptPageReload = () => {
-    cy.intercept('GET', 'api/v2/orgs').as('getOrgs')
-    cy.intercept('GET', 'api/v2/flags').as('getFlags')
-    cy.intercept('GET', 'api/v2/quartz/accounts/**/orgs').as('getQuartzOrgs')
-  }
-
-  const mockQuartzOutage = () => {
-    const quartzFailure = {
-      statusCode: 503,
-      body: 'Service Unavailable',
-    }
-
-    cy.intercept('GET', 'api/v2/quartz/identity', quartzFailure).as(
-      'getQuartzIdentity'
-    )
-    cy.intercept('GET', 'api/v2/quartz/accounts', quartzFailure).as(
-      'getQuartzAccounts'
-    )
-  }
-
```

Comment:
This test is a relic of a time when you could log into cloud via IDPE, but quartz failed to respond. 

That's no longer possible: with cloud login controlled by `/identity`, a failure to load identity means that the user ends up on the login page.